cds_ils/importer/providers/safari/rules/document.py

```diff
@@ -38,17 +38,17 @@ def agency_code(self, key, value):
     return value
 
 
-@model.over("authors", "(^1001)|(^7001)")
+@model.over("authors", "(^1001)|(^1000)|(^7001)")
```

Comment:
```suggestion
@model.over("authors", "(^100)|(^7001)")
```
will cover all the variations of ind1 for key starting with 100

Reply:
What if there is a `1002`? Anyway, fixed it.

Reply:
it is the same field still, I think safari started to use the indicators internally for something, however the field is still author, which is what we want to cover